<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>RecvBandwidthWidget</class>
 <widget class="QWidget" name="RecvBandwidthWidget">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>400</width>
    <height>300</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <item>
    <layout class="QHBoxLayout" name="horizontalLayout">
     <item>
      <layout class="QVBoxLayout" name="controlLayout">
       <item>
        <layout class="QHBoxLayout" name="protocolLayout">
         <item>
          <widget class="QLabel" name="protocolLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>29</height>
            </size>
           </property>
           <property name="text">
            <string>Протокол: </string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QComboBox" name="protocolComboBox"/>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="portLayout">
         <item>
          <widget class="QLabel" name="portLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>29</height>
            </size>
           </property>
           <property name="text">
            <string>Порт:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="portSpinBox">
           <property name="maximum">
            <number>65535</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="threadsLayout">
         <item>
          <widget class="QLabel" name="threadsLabel">
           <property name="maximumSize">
            <size>
             <width>16777215</width>
             <height>29</height>
            </size>
           </property>
           <property name="text">
            <string>Число потоков:</string>
           </property>
          </widget>
         </item>
         <item>
          <widget class="QSpinBox" name="threadsSpinBox">
           <property name="maximum">
            <number>9999</number>
           </property>
          </widget>
         </item>
        </layout>
       </item>
       <item>
        <layout class="QHBoxLayout" name="affinityLayout">
         <item>
          <widget class="QCheckBox" name="affinityCheckBox">
           <property name="text">
            <string>Привязка к процессору</string>
           </property>
          </widget>
         </item>
        </layout>
       </item>
      </layout>
     </item>
     <item>
      <layout class="QVBoxLayout" name="graphicLayout"/>
     </item>
    </layout>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
